release.mk

```diff
@@ -39,6 +39,9 @@ ENABLE_DOCS_PACKAGE := n
 # Whether to build and upload IDE
 ENABLE_IDE := y
 
+# Whether to build or download GNU IDE Plugins
+ENABLE_IDE_PLUGINS_BUILD := y
+
```

Comment:
Please document new option in https://raw.githubusercontent.com/foss-for-synopsys-dwc-arc-processors/toolchain/arc-dev/doc/maintainer/release.rst